assets/css/style.css

```diff
@@ -0,0 +1,208 @@
+h1,h2,h3,h4,h5,h6{
+    text-align: center;
+}
+a{
+    color: black;
+}
+a:hover{
+    color: goldenrod;
+}
+body{
+    background-color: aquamarine;
+    font-family: "Helvetica", "Tahoma", sans-serif;
+}
+main{
+    width: 1000px;
+    margin:0 auto;
+    background-color: white;
+    box-shadow: 0 0 10px black;
+    padding: 20px;
+    box-sizing: border-box;
+}
+ul{
+    list-style: none;
+    padding: 0;
+}
+nav#main-menu>ul{
+    display: flex;
+    justify-content: center;
+    text-transform: uppercase;
+}
+nav#main-menu>ul>li{
+    border-right: 1px solid black;
```

Comment:
use the same sequences everywhere I mean, if you choose 1px solid #000 for border - write 1px solid #000 for border everywhere